CONTRIBUTING.md

```diff
@@ -95,6 +97,43 @@ The non-English languages will fallback to English until they are translated.
 The `src/translations/en.json` will be sent to our translation team and returned for each language we support.
 When a new file is received from the translators, please update all `src/translations/*.json` and `src/translations/*.ts` files with the updated strings.
 
+## Documentation
+
+### Components
```

Comment:
Add it under Documentation because it's more instructional than it is a best practice.